TombLib/TombLib/LevelData/IO/Prj2Writer.cs

```diff
@@ -35,7 +35,7 @@ public static void SaveToPrj2(string filename, Level level, Filter filter = null
                 var tempName = filename + ".tmp";
                 if (File.Exists(tempName)) File.Delete(tempName);
 
-                stream.Seek(0, SeekOrigin.Begin);
+                //stream.Seek(0, SeekOrigin.Begin);
```

Comment:
Why is this not needed anymore exactly? Please consider removing it completely if it's actually pointless.